server.js

```diff
@@ -2,36 +2,80 @@ import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
+import booksData from './data/books.json'
+import Book from './model/book'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-books"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+
+
+if (process.env.RESET_DATABASE) {
+  console.log('Resetting database')
+
+  const seedDatabase = async () => {
+      await Book.deleteMany()
+      await booksData.forEach((book) => new Book(book).save())
+  }
+  seedDatabase()
+}
+
+// PORT
 const port = process.env.PORT || 8080
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
+
+// MIDDLEWARES
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+
+// DISCONNECTED DATABASE
+app.use((req, res, next) => {
+  if (mongoose.connection.readyState === 1) {
+    next()
+  } else {
+    res.status(503).json({ error: 'Service unavailable' })
+  }
+})
+
+
+// ROUTES
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send(`
+  <h1>500 book reviews</h1>
+  <h3>Read moore...</h3>
+  <li>/books</li>
+  <li>/books/autors/:authors</li>
+  `)
 })
```

Comment:
Nice with documentation! ⭐ Consider checking out this npm package: [https://www.npmjs.com/package/express-list-endpoints](https://www.npmjs.com/package/express-list-endpoints)